src/cider/nrepl/middleware/track_state.clj

```diff
@@ -237,7 +238,10 @@
   (future
     (into #{}
           (comp (filter misc/jar-file?)
-                (map #(JarFile. (io/as-file %)))
```

Comment:
It looked like this could have been fixed at Orchard level instead (i.e. making `misc/jar-file?` smarter), but that turned out to have a lot of implications / broken tests.